README.md

```diff
@@ -7,59 +7,684 @@
+including all fulfilled transfers and incoming prepared transfers but not
+including outgoing prepared transfers.
+
+#### Returns
+
+- `return` (Integer String) maximum balance.
+
+-------
+
+### `async TransferLog.getMinimum ()`
+
+Get the TransferLog's minimum balance. This is as low as the balance can go,
+including all fulfilled transfers and outgoing prepared transfers but not
+including incoming prepared transfers.
+
+#### Returns
+
+- `return` (Integer String) minimum balance.
+
+-------
+
+### `async TransferLog.getBalance ()`
+
+Get the TransferLog's balance, including only fulfilled transfers. This
+function is best used for display purposes only. Validation should be done
+using `getIncomingFulfilled`, `getIncomingFulfilledAndPrepared`,
+`getOutgoingFulfilled`, or `getOutgoingFulfilledAndPrepared`.
+
+#### Returns
+
+- `return` (Integer String) total fulfilled balance.
+
+-------
+
+### `async TransferLog.get (id)`
+
+Get a transfer with info from the transfer log.
+
+#### Parameters
+
+- `id` (UUID String) transfer ID.
+
+#### Returns
+
+- `return` (TransferWithInfo) transfer with additional info.
+- `return.transfer` ([Transfer](https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#class-transfer)) transfer with ID of `id`.
+- `return.isIncoming` (Boolean) set to `true` if the transfer is incoming, and `false` if it's outgoing.
+- `return.state` (String) set to `prepared`, `fulfilled`, or `cancelled`.
+- `return.fulfillment` (String) transfer's fulfillment, present only if the state is `fulfilled`.
+
+-------
+
+### `async TransferLog.prepare (transfer, isIncoming)`
+
+Get a transfer from the transfer log. If the transfer would cause the balance to go over
+the transfer log's maximum or under its minimum, this function will throw an error. If the
+transfer cannot be applied for any other reason, an error will be thrown.
+
+If a transfer with the same ID and the same contents as a previous transfer is added, it
+will return successfully, without modifying the database. If a transfer with the same ID as
+a previous transfer is added with different contents, an error will be thrown.
+
+#### Parameters
+
+- `transfer` ([Transfer](https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#class-transfer)) the transfer to be prepared.
+- `isIncoming` (Boolean) set to `true` if the transfer is incoming, and `false` if it's outgoing.
+
+-------
+
+### `async TransferLog.fulfill (transferId, fulfillment)`
+
+Fulfill a transfer currently in the prepared state. If the transfer's state is
+already `fullfilled`, the function will return without error. If the transfer's
+state is `cancelled`, the function will throw an error. If the transfer's state
+is `prepared`, it will be set to `fulfilled` and the fulfillment will be
+stored.
+
+The fulfillment is not compared against the executionCondition of the transfer,
```

Comment:
This should be emphasized (or changed) because it's not what I would expect it to do given the name